app/src/main/java/com/karrar/movieapp/ui/composable/SeasonItem.kt

```diff
@@ -0,0 +1,101 @@
+package com.karrar.movieapp.ui.composable
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.unit.dp
+import coil.compose.rememberAsyncImagePainter
+import com.karrar.movieapp.ui.ui.theme.*
+import com.karrar.movieapp.ui.ui.theme.specing.SpacingTooSmallHorizontal
+import com.karrar.movieapp.ui.ui.theme.specing.SpacingTooSmallVertical
+
+
+@Composable
+fun SeasonItem(
+    seasonName: String,
+    seasonYear: String,
+    seasonDescription: String,
+    episodeCount: String,
+    seasonImageUrl: String,
+) {
+    Box(
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(144.dp)
+            .padding(horizontal = 16.dp)
+    ) {
```

Comment:
Why do you need  box then row?

Reply:
Because of the strange shape of our item 😅
![image](https://user-images.githubusercontent.com/57456722/225442408-cbd44c3c-2c97-4df5-975a-1fcb5a171a2d.png)